### ISSUE TITLE
attempt to create another docker with the same name leads to inappropriate error 

### ISSUE THREAD

**Issue by yarikoptic** [ReproNim/niceman#69]
so first I have created a resource for docker-container and then tried to create yet another one with the same name -- expected some error saying that I have already one like that, but nope... might relate to some stale configuration of mine, but just wanted to check if you don't see that too @mjtravers 

```shell
(git)hopa:~/proj/repronim/niceman[nf-git-trace]
$> niceman create                      
Enter a resource name: docker-smth
 
Enter a resource type [docker-container]: 

...
2017-03-08 11:18:37,343 [INFO   ] Created the environment docker-smth 

$> niceman create
Enter a resource name: docker-smth

2017-03-08 11:18:46,057 [ERROR  ] No section: u'docker' [ConfigParser.py:items:642] (NoSectionError) 

$> cat ~/.config/niceman/inventory.json
{"docker-smth": {"status": "running", "type": "docker-container", "name": "docker-smth", "id": "6a55ccf04897314fd568b74d5ef573755d86b36493d8627953f5624f77df5d7d"}}%                  

$> docker ps
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
6a55ccf04897        ubuntu:latest       "/bin/bash"         36 seconds ago      Up 35 seconds                           docker-smth
```

**Comment by yarikoptic** [comment]
well -- indeed was due to my stale niceman.cfg but we should make operation more robust and if something is not found and we need to crash -- give more meaningful message

**Comment by mjtravers** [comment]
fixed in commit 309bdbbe
